### PR TITLE
[NEW] Add collapsable feature to quotes

### LINKED ISSUE #1689: [BUG] Android app - only part of message is visible
I have incoming webhook UptimeRobot integration.

In official android app is only part of message and it is not possible to read the whole text:

screenshot_2018-09-19-04-36-34-599_chat rocket android

In Rocket.Chat.ReactNative is everything ok:

screenshot_2018-09-19-04-36-41-457_chat rocket reactnative

It is very annoing.

Please fix it. 

```
Rocket.Chat.Android version: 2.6.0
Rocket.Chat.ReactNative version: 1.2.1
Rocket.Chat Server version: 0.69.2
```

### PR DESCRIPTION
**Pull request by leonardoaramaki**
@RocketChat/android

Closes #1689, #1076

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -142,6 +142,9 @@
     <string name="msg_message_copied">Nachricht kopiert</string>
     <string name="msg_delete_message">Lösche Nachricht</string>
     <string name="msg_delete_description">Sind Sie sicher, dass Sie diese Nachricht löschen wollen?</string>
+    <!--Add proper translation-->
+    <string name="msg_view_more">view more</string>
+    <string name="msg_view_less">view less</string>
```
Comment (rafaelks) [inline app/src/main/res/values-de/strings.xml:148]: Need to add the `<!-- TODO Add translation -->` comment here, right?

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -139,6 +139,9 @@
     <string name="msg_member_already_added">Ya has seleccionado este usuario</string>
     <string name="msg_member_not_found">Miembro no encontrado</string>
     <string name="msg_channel_created_successfully">Canal creado con éxito</string>
+    <!--Add proper translation-->
+    <string name="msg_view_more">view more</string>
+    <string name="msg_view_less">view less</string>
```
Comment (rafaelks) [inline app/src/main/res/values-es/strings.xml:145]: Need to add the `<!-- TODO Add translation -->` comment here, right?

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -131,6 +131,9 @@
     <string name="msg_send">envoyer</string>
     <string name="msg_delete_message">Supprimer Message</string>
     <string name="msg_delete_description">Êtes-vous sûr de vouloir supprimer ce message</string>
+    <!--Add proper translation-->
+    <string name="msg_view_more">view more</string>
+    <string name="msg_view_less">view less</string>
```
Comment (rafaelks) [inline app/src/main/res/values-fr/strings.xml:137]: Need to add the `<!-- TODO Add translation -->` comment here, right?

File: app/src/main/res/values-hi-rIN/strings.xml [hunk 4]
```diff
@@ -145,6 +145,9 @@
     <string name="msg_member_already_added">आपने पहले से ही इस यूजर को चुन चुके है।</string>
     <string name="msg_member_not_found">सदस्य नहीं मिला</string>
     <string name="msg_channel_created_successfully">चैनल सफलतापूर्वक बनाया गया</string>
+    <!--Add proper translation-->
+    <string name="msg_view_more">view more</string>
+    <string name="msg_view_less">view less</string>
```
Comment (rafaelks) [inline app/src/main/res/values-hi-rIN/strings.xml:151]: Need to add the `<!-- TODO Add translation -->` comment here, right?

File: app/src/main/res/values-ja/strings.xml [hunk 5]
```diff
@@ -132,6 +132,9 @@
     <string name="msg_file_description">ファイルの説明</string>
     <string name="msg_send">送信</string>
     <string name="msg_sent_attachment">添付ファイルを送信しました</string>
+    <!--Add proper translation-->
+    <string name="msg_view_more">view more</string>
+    <string name="msg_view_less">view less</string>
```
Comment (rafaelks) [inline app/src/main/res/values-ja/strings.xml:138]: Need to add the `<!-- TODO Add translation -->` comment here, right?

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -133,6 +133,8 @@
     // TODO: Add proper translation.
     <string name="msg_delete_message">Delete Message</string>
     <string name="msg_delete_description">Are you sure you want to delete this message</string>
```
Comment (rafaelks) [inline app/src/main/res/values-pt-rBR/strings.xml]: We can translate these strings as well, right? 🙈

File: app/src/main/res/values-ru-rRU/strings.xml [hunk 7]
```diff
@@ -130,6 +130,9 @@
     <string name="msg_delete_description">Вы уверены, что хотите удалить это сообщение?</string>
     <string name="msg_channel_name">Название канала</string>
     <string name="msg_search">Поиск</string>
+    <!--Add proper translation-->
+    <string name="msg_view_more">view more</string>
+    <string name="msg_view_less">view less</string>
```
Comment (rafaelks) [inline app/src/main/res/values-ru-rRU/strings.xml:136]: Need to add the `<!-- TODO Add translation -->` comment here, right?

File: app/src/main/res/values-uk/strings.xml [hunk 8]
```diff
@@ -129,6 +129,9 @@
     <string name="msg_send">Надіслати</string>
     <string name="msg_delete_message">Видалити повідомлення</string>
     <string name="msg_delete_description">Ви впевнені, що хочете видалити це повідомлення?</string>
+    <!--Add proper translation-->
+    <string name="msg_view_more">view more</string>
+    <string name="msg_view_less">view less</string>
```
Comment (rafaelks) [inline app/src/main/res/values-uk/strings.xml:135]: Need to add the `<!-- TODO Add translation -->` comment here, right?

**Comment by rafaelks** [comment]
@leonardoaramaki On scrolling, I loose the button to "view more" or "view less" after I collapse it.

**Comment by leonardoaramaki** [comment]
@rafaelks Indeed, the button vanishes. Will take a look

**Review by luciofm — APPROVED** [review]

**Pull request merged**